.github/workflows/ci.yml

```diff
@@ -18,22 +18,10 @@ jobs:
           node-version: 14
           registry-url: 'https://registry.npmjs.org/'
           scope: '@talend'
-
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - name: Cache node modules
-        uses: actions/cache@v1
-        id: rfd-yarn-cache # use this to check for `cache-hit` (`steps.rfd-yarn-cache.outputs.cache-hit != 'true'`)
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          cache: 'yarn'
 
       - name: Install
-        run: yarn install --frozen-lockfile
+        run: yarn install
```

Comment:
Why remove the `--frozen-lockfile`?

Reply:
lets revert that. The difference is if you change the package.json without re-install (update your lockfile) it will throw exception